torch/overrides.py

```diff
@@ -447,7 +447,11 @@ def get_testing_overrides() -> Dict[Callable, Callable]:
         torch.addr: lambda input, vec1, vec2, beta=1, alpha=1, out=None: -1,
         torch.affine_grid_generator: lambda theta, size, align_corners: -1,
         torch.all: lambda input, dim=None: -1,
-        torch.allclose: lambda input, other, trol=1e-05, atol=1e-08, equal_nan=False: -1,
+        torch.allclose: lambda input,
+        other,
+        trol=1e-05,
+        atol=1e-08,
+        equal_nan=False: -1,
         torch.alpha_dropout: lambda input, p, train, inplace=False: -1,
```

Comment:
The ruff formatted code doesn't look good. We may need to add parentheses.

Reply:
Someone submitted a PR for it https://github.com/astral-sh/ruff/pull/8465 but it seems to have died

Reply:
I add a `# fmt: skip` pragma.